Classes/Private/Manager/Database/OCTRealmManager.h

```diff
@@ -51,6 +53,7 @@
 
 - (OCTFriend *)friendWithFriendNumber:(OCTToxFriendNumber)friendNumber;
 - (OCTChat *)getOrCreateChatWithFriend:(OCTFriend *)friend;
+- (OCTCall *)getOrCreateCallWithFriendNumber:(OCTToxFriendNumber)friendNumber;
```

Comment:
I think that you can leave only method with chat here. And in submanager add private helper method to create call from friendNumber